src/Strategy/Go/Gomod.hs

```diff
@@ -396,6 +397,14 @@ analyze' file = do
     pure ()
   pure (graph, Partial)
 
+-- | This variant of analyze will not attempt to fill in transitive dependencies.
+analyzeStatic :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> m (Graphing Dependency, GraphBreadth)
```

Comment:
If this isn't in the user facing docs already please add it!

Reply:
Good idea, I've added it.